src/cuttlefish_escript.erl

```diff
@@ -234,20 +223,45 @@ describe_doc(Match, FindResults) ->
                 [?STDOUT("    ~s", [cuttlefish_variable:format(S)]) || S <- See]
         end.
 
-describe_default(Match) ->
+describe_default(Type, Default) ->
     ValidValues = [
         ?FORMAT("~n     - ~s", [cuttlefish_conf:pretty_datatype(Type)]) ||
-        Type <- lists:flatten([cuttlefish_mapping:datatype(Match)]) ],
+        Type <- lists:flatten([Type]) ],
     ?STDOUT("   Valid Values: ~s", [ValidValues]),
-    case cuttlefish_mapping:has_default(Match) of
-        true ->
+    case Default of
+        undefined ->
+            ?STDOUT("   No default set", []);
+        _ ->
             ?STDOUT("   Default Value : ~s",
-                [format_datatype(cuttlefish_mapping:default(Match),
-                    cuttlefish_mapping:datatype(Match))]);
-        false ->
-            ?STDOUT("   No default set", [])
+                [format_datatype(Default, Type)])
     end.
 
+fetch(ParsedArgs, [Query|_]) when is_list(Query) ->
```

Comment:
I think it's better if we can push the opaque `ParsedArgs` arg to higher level.
So we can expose fetch/2 to external. e.g. `fetch(ConfFile, SchemaFile, Query)`
Then later we could also cache `load_schema` result in a ETS table for faster query.

Reply:
and this API is better to be implemented somewhere else.
`cuttlefish_escript` is not an API module.